src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { regexpAsciiEmail, regexpUnicodeEmail } from './regexp';
+
+export const validateEmail = (input: unknown, unicode = true): boolean => {
```

Comment:
Because following `typeof input === 'string'` check gets alert. The check is required for pure JavaScript developers.

Reply:
I think you should use both type checking with `typeof` and the TypeScript type because what `input` requires `unknown` even though it actually takes `string` is unkind for TypeScript users.